app/models/json_builders/channels_json_builder_v2.rb

```diff
@@ -53,13 +54,19 @@ def include_verified_channel(verified_channel)
     return if @excluded_channel_ids.include?(verified_channel.details.channel_identifier)
     # Skip if channel publisher has not yet KYC'd
     return unless verified_channel.publisher&.uphold_connection&.is_member
+
+    wallet_address_id = verified_channel.publisher&.uphold_connection&.address
+    return if wallet_address_id.nil?
+    return if existing_wallet_address?(wallet_address_id)
```

Comment:
Don't we want to push the channel even if the wallet address is existing?

Reply:
It's a hash to check to see if the address is a duplicate.